packages/odyssey-storybook/src/components/odyssey-mui/TextField/TextField.stories.tsx

```diff
@@ -88,19 +105,44 @@ export default {
 const Template: Story = (args) => {
   return (
     <FormControl disabled={args.disabled} error={args.invalid}>
-      <InputLabel id="demo-text-field-label">{args.label}</InputLabel>
+      <InputLabel id="demo-text-field-label">
+        {args.label}
+        {!args.required && (
+          <Typography variant="subtitle1">{args.optionalLabel}</Typography>
+        )}
+      </InputLabel>
       {args.hint && (
         <FormHelperText id="textfield-hint">{args.hint}</FormHelperText>
       )}
-      <OutlinedInput
-        aria-describedby="textfield-hint textfield-error"
+      <InputBase
+        inputProps={{ "aria-describedby": "textfield-hint textfield-error" }}
```

Comment:
Putting in an example where we're adding an object that's not memoized could be an issue, but it's probably fine here.